apps/web/src/stores/projects.ts

```diff
@@ -21,13 +23,13 @@ export default function useProjects(opts?: SWRConfiguration) {
     if (!data) return []
 
     return data
-  }
+  }, [toast])
```

Comment:
useless toast dependency atm

Reply:
Why?

Reply:
because you are not using the toast in the function

Reply:
Ok but what if I would be using it? I'm asking because this function is defined at module level but exported from `useToast` hook 
https://github.com/latitude-dev/latitude-llm/blob/main/packages/web-ui/src/ds/atoms/Toast/useToast.ts#L141-L168

Reply:
oh wow didn't notice that, yeah then i think you never need to put toast as dependency since it will never change anyways